datahub_sap_hana/ingestion.py

```diff
@@ -178,3 +200,115 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_column_lineage_view_definitions(self) -> Iterable[Tuple[str, str]]:
+
+        url = self.config.get_sql_alchemy_url()
+        engine = create_engine(url, **self.config.options)
+
+        with engine.connect() as conn:
+            inspector = inspect(conn)
+            schema = inspector.get_schema_names()  # returns a list
+            views = inspector.get_view_names(
+                schema[0])  # returns a list
+
+            for view in views:
+                definition = inspector.get_view_definition(view, schema[0])
+                yield (view, definition)
+                # print(view, definition)
+
+    def get_lineage_for_column(self, view_name: str, view_sql: str) -> Iterable[Tuple[str, Any]]:
```

Comment:
I am not sure this function is named correctly. I think `_get_column_lineage_for_view` make more sense given the input arguments. 

Also, I think this is one where we want to include a doc string about what is happening.